memote/support/basic.py

```diff
@@ -273,6 +273,36 @@ def find_duplicate_metabolites_in_compartments(model):
     return duplicates
 
 
+def find_duplicate_reactions(model):
+    """
+    Return list of reactions with duplicates.
+
+    All models should contain a unique set of reactions. This function checks
+    for and returns a list of tuples containing the duplicate reactions. An
+    example of this would be finding reactions PGI1 and PGI2, both of which
+    convert G6P(c) to F6P(c) in the same compartments.
+
+    Parameters
+    ----------
+    model : cobra.Model
+        The metabolic model under investigation
+
+    """
+    duplicates = []
+    RXN_DB = ["metanetx.reaction", "kegg.reaction", "brenda", "rhea", "biocyc",
```

Comment:
All capital variable names are reserved for global variables. How about `rxn_db_identifiers`?